emcc.py

```diff
@@ -3475,6 +3475,9 @@ def consume_arg_file():
         # set the emscripten debug level to 3 so that we do not remove that
         # debug info during link (during compile, this does not make a
         # difference).
+        else:
+          # Assume unknown non-integer levels are clang flags that emit dwarf.
+          settings.GENERATE_DWARF = 1
```

Comment:
Why is this needed?

Reply:
Not sure why it is needed. Will remove it for now. @dschuff, feel free to add it back later if necessary.

Reply:
I don't know why this is needed either, and not sure what to do with this. (And I don't think tests here cover this) Leave it or delete it?

Reply:
I think this is here to cover flags such as `-gdwarf-5` which causes DWARF to be emitted and also causes DWARF5 to be used. Maybe I should add a test for that.

Reply:
Oh actually the comment just above this `else` clauses also describes this use case. It's weird that they are separated, I'll consolidate them.